tests/checkPot.py

```diff
@@ -120,8 +138,9 @@ def checkPot(fileName):
 				sourceLines = []
 				context = ""
 				continue
-			if line == 'msgid ""':
+			if line == 'msgid ""' and not passedHeader:
 				# Header.

```

Comment:
Oh, I see. Each pot file has a header with metadata about the pot file. EG:
```
msgid ""
msgstr ""
"Project-Id-Version: NVDA branch-1d146f6\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-04-28 15:40+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
```
Since there is only one header in the file, we don't need to reset passedHeader. This metadata is not used by checkPot and thus discarded. 

A comment would help here. For instance "There is a single empty msgid for the pot file metadata header (contained in the msgstr portion). It is expected at the start of the file, after passing it, all subsequent entries should be considered."

Reply:
Done:  d19e8d249